text-generation-inference/server/text_generation_server/jetstream_pt_support/engine_loader.py

```diff
@@ -18,17 +18,29 @@
     from transformers import PretrainedConfig
 from transformers import AutoConfig
 
-from .llama_model_exportable_hf import TransformerHf
+from .gemma_model_hf import GemmaModelHf as GemmaModel
+from .llama_model_exportable_hf import TransformerHf as LlamaModel
 
 
-def load_llama_model_info(config: "PretrainedConfig") -> Any:
+def _get_head_dim(config: "PretrainedConfig") -> int:
+    if hasattr(config, "head_dim"):
+        return config.head_dim
+    return config.hidden_size // config.num_attention_heads
```

Comment:
You should be able to use `optimum.NormalizedConfig` which does provide this oob

Reply:
umh this didn't work as I expected, maybe I am doing something wrong? I also see we have conditional code for this, see here:
https://github.com/huggingface/optimum/blob/main/optimum/onnxruntime/modeling_decoder.py#L339..L342